src/exposed.tsx

```diff
@@ -1,5 +1,6 @@
 import { DndProvider } from "react-dnd";
-import { HTML5Backend } from "react-dnd-html5-backend";
```

Comment:
remove/replace also in other components.  e.g.: `src/admin/components/SchemaWizard.jsx`